### ISSUE TITLE
[Feature Request] ROM Shortcut Widget Compatibility

### ISSUE THREAD

**Issue by Ratlover14** [citra-emu/citra-android#278]
On most, if not all modern Android phones, widgets are available. Some can act as shortcuts as well. Well, why not have a shortcut widget that jumps straight to opening a ROM file? For example, MelonDS, a DS emulator, has this feature. After finding the widget, it opens a menu where you select which game you want it to short cut to. Then the widget is on your home screen, and you can just click it to launch the game, saving some time of opening the app.
I want this feature because I'm rethemeing my phone like a 3Ds, and this would be like a finishing touch. But it could also be a great feature for other reasons :)
I've also attached images of the MelonDS widget process

TL;DR: Add ROM shortcut widget compatibility to make it easier and faster to open games
![Screenshot_20220721-123633_One UI Home](https://user-images.githubusercontent.com/109756507/180277102-8651520f-e6cf-4a56-bb26-65529bfac0fc.jpg)
![Screenshot_20220721-123642_melonDS](https://user-images.githubusercontent.com/109756507/180277104-14f41040-fa3c-40f0-9f7c-fc79bc96e7c9.jpg)
![Screenshot_20220721-133556_One UI Home](https://user-images.githubusercontent.com/109756507/180277106-8bf158db-9fda-45ed-84e7-e58913fd9554.jpg)
![Screenshot_20220721-133759_One UI Home](https://user-images.githubusercontent.com/109756507/180277449-7bba6e9e-9c4d-4cc0-a332-ecb879b0b4f6.jpg)
